build.sc

```diff
@@ -1,12 +1,20 @@
-import $ivy.`de.tototec::de.tobiasroeser.mill.integrationtest_mill0.9:0.4.0`
+import $ivy.`de.tototec::de.tobiasroeser.mill.integrationtest_mill0.9:0.4.1-30-f29f55`
 import de.tobiasroeser.mill.integrationtest._
 import mill._
 import mill.scalalib._
+import mill.scalalib.api.Util.scalaNativeBinaryVersion
 import publish._
 
-object tpolecat extends ScalaModule with PublishModule {
-  def scalaVersion = "2.13.4"
-  def artifactName = "mill-tpolecat"
+val millVersions                           = Seq("0.10.0", "0.9.12")
```

Comment:
It's better to depend on `0.9.3` than `0.9.12` to avoid using potentially newly introduces API.